### PR TITLE
Various small things

### PR DESCRIPTION
**Pull request by jsgoyette**
This pull request covers the various small issues described in issues 2-4.

### CODE REVIEW

File: src/Bitfinex.coffee [hunk 1]
```diff
@@ -9,7 +9,8 @@ qs = require('qs')
 #
 module.exports = class Bitfinex
 	constructor: (@key, @secret) ->
-		@url = 'https://api.bitfinex.com/v1/'
+		@version = '/v1/'
+		@url = 'https://api.bitfinex.com' + @version
```
Comment (lookfirst) [inline src/Bitfinex.coffee]: Use string interpolation here. =)

@url = "https://api.bitfinex.com#{@version}"

**Comment by lookfirst** [comment]
Thanks for the fixes... just fix the thing above and we are good to go. =)

**Comment by jsgoyette** [comment]
Alright, I added a new commit to use string interpolation. I don't use coffee very much so I forget about some of those finer points (even though string interpolation was used multiple times in the code!).

BTW, thanks for your work in building this API wrapper...it's really handy!

**Comment by lookfirst** [comment]
Sweet. Thanks for your fast response and I'm glad you enjoyed it. I ended up never really putting it into production so I didn't get a chance to fully test it out after I created it. I'm glad you're getting some use out of it now! If you have any other PR's, I'm happy to merge them in as fast as you can submit them. Heh.

**Pull request merged**